app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  before_filter :authenticate_user!
+
+  def delete
+  end
+
+  def edit_password
+    @user = current_user
+  end
+
+  def update_password
+    @user = User.find(current_user.id)
+    valid_password = @user.valid_password?(current_password)
+    if valid_password && @user.update(user_params)
+      sign_in(@user, bypass: true)
+      redirect_to(profile_path, notice: 'Your password was successfully updated')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.